src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1880,5 +1882,15 @@ private IOperation CreateBoundRangeVariableOperation(BoundRangeVariable boundRan
             // We do not have operation nodes for the bound range variables, just it's value.
             return Create(boundRangeVariable.Value);
         }
+
+        private IOperation CreateDiscardExpressionOperation(BoundDiscardExpression boundNode)
+        {
+            return new DiscardOperation((IDiscardSymbol)boundNode.ExpressionSymbol,
```

Comment:
Nit: I assume the plan is to generate this file too (at some point). If so, maybe match the formatting in the rest of the file, so ease that transition.

Reply:
That was the plan at one point, but there are too many special cases when translating from BoundNodes to IOperation that we're never going to be able to generate this file.

---
In reply to: [178197139](https://github.com/dotnet/roslyn/pull/25820#discussion_r178197139) [](ancestors = 178197139)